test/download.jl

```diff
@@ -1,6 +1,6 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
-let urls = ["http://httpbin.org/ip", "https://httpbin.org/ip"]
+let urls = ["http://httpbin.julialang.org/ip", "https://httpbin.julialang.org/ip"]
```

Comment:
Looks like we only get `https` here.

Reply:
```suggestion
let urls = ["http://julialang.org", "https://httpbin.julialang.org/ip"]
```

Reply:
Also needs to be adjusted below though.

Reply:
I'm not sure what you're saying here; this chunk is only testing that `Base.download_url()` is affected by `Base.DOWNLOAD_HOOKS`; nothing is actually downloaded.

Reply:
ah, my bad.

Reply:
I've added a comment to make this more explicit.